site/content/blog/markdowndb-basics-tutorial-2023.md

```diff
@@ -0,0 +1,39 @@
+# Learn how to use MarkdownDB with our First Tutorial!
+
+![Gif](/blog/markdowndb.gif)
+
+We've just released our first tutorial that covers the fundamentals of [MarkdownDB](https://github.com/datopian/markdowndb) - our new package for treating markdown files as a database. If you've been curious about how to manage your markdown files more effectively, this tutorial is an excellent starting point!
+
+[👉 Check it out!](https://github.com/datopian/markdowndb/blob/main/examples/basic-example/README.md)
+
+## What is MarkdownDB?
+
+MarkdownDB can parse your markdown files, extract structured data, such as frontmatter, tags, back- and forward links, and create an index in a local SQLite database. It also provides a lightweight JavaScript API for querying the database and importing files into your application. With MarkdownDB, you have a powerful tool that allows you to efficiently query your markdown data.
+
+[🔍 Click here to learn more!](https://github.com/datopian/markdowndb)
+
+## What you're going to learn
+
+This tutorial guides you through the steps of creating SQLite database of our GitHub projects, using markdown files and MarkdownDB package. In addition, it demonstrates how to write a simple script to query a database for the projects and display them in an attractive format on the terminal.
+
+Key highlights from the tutorial include:
+
+1. Creating markdown files with some basic information for each GH project.
+
+2. Indexing markdown files into an SQLite database.
+
+3. Exploring the SQLite database using SQLite viewer or in the command line.
+
+4. Querying the database in a simple Node.js app and displaing them in the terminal.
+
+5. Adding metadata, like name, numbers of stars or number of forks, to project files.
+
+6. Pretty printing the output in the terminal.
+
+By the end of the tutorial, you'll have a simple catalog of a few of our GitHub projects using markdown files and the MarkdownDB package.
+
```

Comment:
Would there be any harm in bulk inlining the tutorial here? I think we may as well ...

(I actually think tutorials as blog posts is a good idea - at least in first instance ...)

Reply:
I thought it was supposed to be a "micro post", sth like the announcement on Twitter but in form of a blog post